src/consensus/blockverify.cpp

```diff
@@ -0,0 +1,162 @@
+// Copyright (c) 2009-2010 Satoshi Nakamoto
+// Copyright (c) 2009-2014 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include "consensus/consensus.h"
+
+#include "arith_uint256.h"
+#include "consensus/validation.h"
+#include "primitives/block.h"
+#include "tinyformat.h"
+
+#include <algorithm>  
+
+static const unsigned int MEDIAN_TIME_SPAN = 11;
```

Comment:
Feel free to suggest something. Probably better to comment in the smaller PRs for now as this branch will likely require more rebases and force pushes. In this case, the relevant PR is #6009